### ISSUE TITLE
Page for Sign Up or Log In user flow capitalizes first letter of email address

### ISSUE THREAD

**Issue by TomSoderling** [microsoft/appcenter#1067]
**Describe the solution you'd like**
When entering the email address to sign up, the text box capitalizes the first letter of the email address and it's really hard to correct it. The text box should just take whatever input the user types in - don't try to auto correct it.

**Describe alternatives you've considered**
Deleting the first character and re-entering it doesn't work - it just re-capitalizes it
Typing a letter next to the one to delete, then deleting the capital seems to work sometimes.

**Additional context**
![Simulator Screen Shot - iPhone Xʀ - 2019-09-21 at 15 06 44](https://user-images.githubusercontent.com/18268845/65378717-fde44c00-dc81-11e9-95c0-359bf8c4a24d.png)

**Comment by amchew** [comment]
Hey @TomSoderling, thanks for reporting this! A question, although it the first alphabet is capitalized, does the reset password flow still work?

**Comment by TomSoderling** [comment]
Unsure. If I attempt to Sign In and tap the "forgot your password" link without entering anything on that page, I'm immediately navigated back to the app with an exception: "access_denied;AADB2C90118: The user has forgotten their password"
It gives a correlation ID and timestamp as well

**Comment by Zakeelm** [comment]
@TomSoderling  Thank you so much for your feedback. We announced this week in a [blog post]( https://devblogs.microsoft.com/appcenter/app-center-mbaas-retirement/) that we’re retiring the Auth and Data services later this year and recommending customers transition to Azure Active Directory B2C and Azure Cosmos DB instead. Let me know if you have any additional questions about this transition and i'll make sure to get them answered! Closing this issue.
